internal/pkg/print/print.go

```diff
@@ -37,7 +38,7 @@ type Printer struct {
 
 // Creates a new printer, including setting up the default logger.
 func NewPrinter() *Printer {
-	logger := slog.New(slog.NewTextHandler(os.Stderr, &slog.HandlerOptions{AddSource: false, Level: slog.LevelDebug}))
+	logger := slog.New(tint.NewHandler(os.Stderr, &tint.Options{AddSource: false, Level: slog.LevelDebug}))
```

Comment:
Maybe we need to also check if Windows is being used and adapt like described here? https://github.com/lmittmann/tint?tab=readme-ov-file#windows-support

Reply:
check the `openBrowser` method for a `switch...case` on OS

Reply:
Great catch, thank you! I investigated the [go-colorable](https://github.com/mattn/go-colorable) they suggest in the documentation and its fairly simple, should be ok to add.
There's no need for a `switch...case` though, the go-colorable package already handles other OSes by [not doing anything](https://github.com/mattn/go-colorable/blob/master/colorable_others.go) if it isn't Windows.

Reply:
It's also used by the GitHub cli, so we have more assurance there